doc/src/manual/networking-and-streams.md

```diff
@@ -184,6 +185,23 @@ julia> open("hello.txt") do f
 "HELLO AGAIN."
 ```
 
+If you want to redirect stdout to a file
+
+```# Open file for writing
+out_file = open("output.txt", "w")
+
+# Redirect stdout to file
+stdout = out_file
```

Comment:
This does not redirect `stdout` in code following this, that already uses the `stdout` variable internally. It's just rebinding the variable here. Redirecting `stdout` to a file is done like so:

```
redirect_stdout(open("output.txt")) do
    # your code here
end
```

Reply:
Moreover, `redirect_stdout` also redirects standard output from things like C libraries that don't use the Julia `stdout` variable.